src/HealthKit/Enums.cs

```diff
@@ -6,6 +6,7 @@
 namespace XamCore.HealthKit
 {
 	// NSInteger -> HKDefines.h
+	[Watch (2,0)]
```

Comment:
not a big deal, but for the future, we don't need `[Watch (2,0)]` attributes since it's the first supported version, i.e. if the API is present without availability attributes it's assumed it was always available (and watchOS 1.x did not have native apps so there's no confusion)